ms/MSOper/MSMetaData.cc

```diff
@@ -3571,9 +3913,22 @@ void MSMetaData::_getScanAndSubScanProperties(
     }
     SHARED_PTR<std::map<SubScanKey, SubScanProperties> > myssprops;
     SHARED_PTR<std::map<ScanKey, ScanProperties> > myscanprops;
+#ifdef _OPENMP
+    if (_parallel) {
+        _computeScanAndSubScanPropertiesParallel(
+            myscanprops, myssprops, showProgress
+        );
+    }
+    else {
+        _computeScanAndSubScanProperties(
+            myscanprops, myssprops, showProgress
+        );
+    }
+#else
     _computeScanAndSubScanProperties(
         myscanprops, myssprops, showProgress
     );
+#endif
```

Comment:
is this branch necessary? is there a reason to not always take the parallel code path and simply limit the openmp threads to 1 if `_parallel` is false

this is done via

```
#pragma omp parallel for if (_parallel)
...
```